examples/convert_eeg_to_bids.py

```diff
@@ -110,85 +152,72 @@
 print(write_raw_bids.__doc__)
 
 ###############################################################################
-# We loaded 'S001R02.edf', which corresponds to subject 1 in the second task.
-# The second task was to rest with closed eyes.
-subject_id = '001'  # zero padding to account for >100 subjects in this dataset
-task = 'resteyesclosed'
-raw_file = raw
-bids_root = os.path.join(mne_data_dir, 'eegmmidb_bids')
+# We loaded ``S001R02.edf``, which corresponds to subject 1 in the second run.
+# In the second run of the experiment, the task was to rest with closed eyes.
+
+# zero padding to account for >100 subjects in this dataset
+subject_id = '001'
+
+# define a task name and a directory where to save the data to
+task = 'RestEyesClosed'
+bids_root = os.path.join(mne_data_dir, 'eegmmidb_bids_eeg_example')
+
+# Start with a clean directory in case the directory existed beforehand
+sh.rmtree(bids_root, ignore_errors=True)
 
 ###############################################################################
 # Now we just need to specify a few more EEG details to get something sensible:
 
 # Brief description of the event markers present in the data. This will become
-# the `trial_type` column in our BIDS `events.tsv`. We know about the event
-# meaning from the documentation on PhysioBank
-trial_type = {'rest': 0, 'imagine left fist': 1, 'imagine right fist': 2}
+# the ``trial_type`` column in our BIDS ```events.tsv``.
+# From the online documentation of our downloaded data we know that in the
+# rest "eyes closed" task, there is only a single event marker: "0"
+event_id = {'rest': 0}
 
 # Now convert our data to be in a new BIDS dataset.
 bids_basename = make_bids_basename(subject=subject_id, task=task)
-write_raw_bids(raw_file, bids_basename, bids_root, event_id=trial_type,
+write_raw_bids(raw, bids_basename, bids_root, event_id=event_id,
                events_data=events, overwrite=True)
-###############################################################################
-# What does our fresh BIDS directory look like?
-print_dir_tree(bids_root)
 
 ###############################################################################
-# Looks good so far, let's convert the data for all tasks and subjects.
```

Comment:
just checking, you deleted this because it's now part of another example?

Reply:
yes!